### PR TITLE
Update `protobuf/README.md`

### PR DESCRIPTION
**Pull request by sijie**
*Problem*:

the cpp protobuf files are automatically generated by Cmake.
we don't need extra instructions for generating cpp files.

*Solution*:

Remove the content about cpp from `protobuf/README.md`

### CODE REVIEW

**Comment by maskit** [comment]
If users who know nothing read the section, they may think "What about C++?" since we mention it at the end of previous section. I think we should probably keep the section and tell the fact that the file will be generated automatically during build.

**Comment by sijie** [comment]
@maskit good suggestion. I updated the README.md to tell the C++ code is automatically generated by 'cmake'

File: protobuf/README.md [hunk 1]
```diff
@@ -17,28 +17,16 @@ To re-generate the `PulsarApi.java` code you need to apply a patch to the protob
...
-autoreconf --install
-./configure
-make
-
-### Re-generate PulsarApi
-cd ${PULSAR_HOME}/pulsar-client-cpp/
-export PROTOC=${HOME}/protobuf/src/protoc 
-./generate_protobuf.sh
-
 ### For Java ###
```
Comment (maskit) [inline protobuf/README.md]: This line is redundant because the whole section is for Java now.

Comment (sijie) [reply protobuf/README.md]: done

**Pull request merged**